internal/definitions/definition_handler.go

```diff
@@ -61,43 +61,43 @@ func NewSystemHandlers(di database.Plugin, dx dataexchange.Plugin, dm data.Manag
 	}
 }
 
-func (sh *systemHandlers) GetGroupByID(ctx context.Context, id string) (*fftypes.Group, error) {
-	return sh.messaging.GetGroupByID(ctx, id)
+func (dh *definitionHandlers) GetGroupByID(ctx context.Context, id string) (*fftypes.Group, error) {
+	return dh.messaging.GetGroupByID(ctx, id)
 }
 
-func (sh *systemHandlers) GetGroups(ctx context.Context, filter database.AndFilter) ([]*fftypes.Group, *database.FilterResult, error) {
-	return sh.messaging.GetGroups(ctx, filter)
+func (dh *definitionHandlers) GetGroups(ctx context.Context, filter database.AndFilter) ([]*fftypes.Group, *database.FilterResult, error) {
+	return dh.messaging.GetGroups(ctx, filter)
 }
 
-func (sh *systemHandlers) ResolveInitGroup(ctx context.Context, msg *fftypes.Message) (*fftypes.Group, error) {
-	return sh.messaging.ResolveInitGroup(ctx, msg)
+func (dh *definitionHandlers) ResolveInitGroup(ctx context.Context, msg *fftypes.Message) (*fftypes.Group, error) {
+	return dh.messaging.ResolveInitGroup(ctx, msg)
 }
 
-func (sh *systemHandlers) EnsureLocalGroup(ctx context.Context, group *fftypes.Group) (ok bool, err error) {
-	return sh.messaging.EnsureLocalGroup(ctx, group)
+func (dh *definitionHandlers) EnsureLocalGroup(ctx context.Context, group *fftypes.Group) (ok bool, err error) {
+	return dh.messaging.EnsureLocalGroup(ctx, group)
 }
 
-func (sh *systemHandlers) HandleSystemBroadcast(ctx context.Context, msg *fftypes.Message, data []*fftypes.Data) (valid bool, err error) {
+func (dh *definitionHandlers) HandleDefinitionBroadcast(ctx context.Context, msg *fftypes.Message, data []*fftypes.Data) (valid bool, err error) {
 	l := log.L(ctx)
 	l.Infof("Confirming system broadcast '%s' [%s]", msg.Header.Tag, msg.Header.ID)
 	switch fftypes.SystemTag(msg.Header.Tag) {
 	case fftypes.SystemTagDefineDatatype:
-		return sh.handleDatatypeBroadcast(ctx, msg, data)
+		return dh.handleDatatypeBroadcast(ctx, msg, data)
 	case fftypes.SystemTagDefineNamespace:
-		return sh.handleNamespaceBroadcast(ctx, msg, data)
+		return dh.handleNamespaceBroadcast(ctx, msg, data)
 	case fftypes.SystemTagDefineOrganization:
-		return sh.handleOrganizationBroadcast(ctx, msg, data)
+		return dh.handleOrganizationBroadcast(ctx, msg, data)
 	case fftypes.SystemTagDefineNode:
-		return sh.handleNodeBroadcast(ctx, msg, data)
+		return dh.handleNodeBroadcast(ctx, msg, data)
 	case fftypes.SystemTagDefinePool:
-		return sh.handleTokenPoolBroadcast(ctx, msg, data)
+		return dh.handleTokenPoolBroadcast(ctx, msg, data)
 	default:
-		l.Warnf("Unknown topic '%s' for system broadcast ID '%s'", msg.Header.Tag, msg.Header.ID)
+		l.Debugf("Unknown topic '%s' for system broadcast or definition ID '%s'", msg.Header.Tag, msg.Header.ID)
```

Comment:
This is really only for definitions now, right? The "or" seems superfluous. Also we say "topic" but actually log the "tag".